pkg/service/rtcservice.go

```diff
@@ -407,6 +411,7 @@ func (s *RTCService) ParseClientInfo(r *http.Request) *livekit.ClientInfo {
 
 	// attempt to parse types for SDKs that support browser as a platform
 	if ci.Sdk == livekit.ClientInfo_JS ||
+		ci.Sdk == livekit.ClientInfo_REACT_NATIVE ||
```

Comment:
Does RN send a user agent too? I think if you are setting OS explicitly, this wouldn't be needed right?